pkg/ruler/ruler.go

```diff
@@ -86,6 +86,8 @@ type Config struct {
 	NotificationQueueCapacity int `yaml:"notification_queue_capacity"`
 	// HTTP timeout duration when sending notifications to the Alertmanager.
 	NotificationTimeout time.Duration `yaml:"notification_timeout"`
+	// Client configs for interacting with the Alertmanager
+	NotifierConfig `yaml:"alertmanager_client"`
```

Comment:
Why is this embedded field instead of named field? Nothing else in this struct is using embedding.